migrations/201811160000-add-data-to-orders.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface
+      .addColumn('Orders', 'data', {
+        type: Sequelize.JSON,
+      })
+      .then(() =>
+        queryInterface.addColumn('OrderHistories', 'data', {
+          type: Sequelize.JSON,
+        }),
+      )
+      .then(() => {
+        console.log('>>> done');
+      });
+  },
+
+  down: (queryInterface, Sequelize) => {
+    return queryInterface
+      .removeColumn('Orders', 'data')
+      .then(() => queryInterface.removeColumn('OrderHistories', 'data'))
+      .then(() => {
+        console.log('>>> done');
```

Comment:
not a big deal but we could remove this console log later!

Reply:
@marcogbarcellos It's a migration

Reply:
you could then improve the message on the console log as when you run the sequelize `db:migrate:undo` it'll run all files and will perhaps not be that clear it's regarding this file.

Reply:
I just did this migration like multiple before. If these `'>>> done'` messages are a problem, then we should remove all of them.